code/modules/power/cell.dm

```diff
@@ -201,7 +201,8 @@
 		return 0
 
 /obj/item/stock_parts/cell/get_part_rating()
-	return rating * maxcharge
+	rating = max(round(maxcharge/8000,1),1)
```

Comment:
I feel like this operating shouldn't be done on the `rating` var? You're mutating the actual rating of the cell whenever you call this.